src/main/java/xdev/ui/laf/NimbusLookAndFeel.java

```diff
@@ -33,8 +33,14 @@ public void setLookAndFeel() throws LookAndFeelException
 	{
 		try
 		{
-			UIManager.setLookAndFeel(com.sun.java.swing.plaf.nimbus.NimbusLookAndFeel.class
-					.getName());
+			if (System.getProperty("java.version").startsWith("1.")) {
+				// Java 8 and below
+				UIManager.setLookAndFeel("com.sun.java.swing.plaf.nimbus.NimbusLookAndFeel");
+			}else {
```

Comment:
Also here the format seems to be wrong.
Please recheck all modified files.

Reply:
Checked and fixed all files 👍